### PR TITLE
Influx config

### PR DESCRIPTION
**Pull request by kelseiv**
- Document new profile (config) management CLI commands.

### CODE REVIEW

File: content/v2.0/reference/cli/influx/config/_index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description: The 'influx config' command and subcommands manage multiple InfluxDB connection configurations.
+menu:
+  v2_0_ref:
+    name: influx config
+    parent: influx
+weight: 101
+v2.0/tags: [config]
+---
+
+The `influx config` command manages multiple InfluxDB connection configurations in the `config` file (by default, stored at `~/.influxdbv2/config`). Each connection includes a url, token, associated organization (`org`), and active setting.
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/_index.md]: - ~url~ -> URL
- ~('org`)~

I'd either make them all inline code using the actual keys from the file, or all regular english.

File: content/v2.0/reference/cli/influx/config/_index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: influx config – Config management commands
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/_index.md]: - ~– Config management commands~

I went through and removed all these "summaries" in the titles on the beta-6 branch. There weren't really adding anything.

File: content/v2.0/reference/cli/influx/config/_index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+## Usage
+```
+influx config [flags]
+influx config [command]
+```
+
+## Subcommands
+| Subcommand                                         | Description   |
+|:----                                               |:-----------   |
+| [create](/v2.0/reference/cli/influx/config/create) | Create a new connection to include in the configuration (`config`) file (by default, at `~/.influxdbv2/config`).
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/_index.md]: Create a new connection configuration

File: content/v2.0/reference/cli/influx/config/create.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+title: influx config create
+description: The 'influx config create' command creates a new InfluxDB connection configuration.
+menu:
+  v2_0_ref:
+    name: influx config create
+    parent: influx config
+weight: 201
+---
+
+The `influx config create` command creates a new InfluxDB connection configuration in the `config` file (by default, stored at `~/.influxdbv2/config`).
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/create.md]: ...creates a new InfluxDB connection configuration **and stores it** in the `config` file (by default, stored at `~/.influxdbv2/config`).

File: content/v2.0/reference/cli/influx/config/list.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+title: influx config list
+description: The 'influx config list' command lists all InfluxDB connection configurations.
+menu:
+  v2_0_ref:
+    name: influx config list
+    parent: influx config
+weight: 201
+---
+
+The `influx config list` command lists all InfluxDB connection configurations in the `config` file (by default, stored at `~/.influxdbv2/config`). Each connection configuration includes a url, authentication token, and active setting. By default, the list includes an asterisk next to the active connection configuration. InfluxDB reads the token from the active connection configuration, so you don't have to manually enter a token to log into InfluxDB.
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/list.md]: - ~By default, the list includes an asterisk next to the active connection configuration.~ -> An asterisk (`*`) indicates the active configuration.
- "InfluxDB reads the token from the active connection configuration, so you don't have to manually enter a token to log into InfluxDB." - This feels out of place here. I think this should be on the `influx config` landing page.

File: content/v2.0/reference/cli/influx/config/_index.md (hunk 1 above)
Comment (kelwang) [inline content/v2.0/reference/cli/influx/config/_index.md]: actually the file stored called `configs` not config

File: content/v2.0/reference/cli/influx/config/_index.md (hunk 2 above)
Comment (kelseiv) [reply content/v2.0/reference/cli/influx/config/_index.md]: good call!

File: content/v2.0/reference/cli/influx/config/list.md (hunk 5 above)
Comment (kelseiv) [reply content/v2.0/reference/cli/influx/config/list.md]: moved last statement to landing page

File: content/v2.0/reference/cli/influx/config/_index.md (hunk 1 above)
Comment (kelseiv) [reply content/v2.0/reference/cli/influx/config/_index.md]: updated filename to `configs`

File: content/v2.0/reference/cli/influx/config/_index.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+## Usage
+```
+influx config [flags]
+influx config [command]
+```
+
+## Subcommands
+| Subcommand                                         | Description   |
+|:----                                               |:-----------   |
+| [create](/v2.0/reference/cli/influx/config/create) | Create a new connection configuration to include in the configuration (`config`) file (by default, at `~/.influxdbv2/config`).
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/_index.md]: This is just a short summary table, so I'd keep these descriptions brief. "Create a new connection configuration" should be fine here. The other content is just details that can be explained elsewhere.

File: content/v2.0/reference/cli/influx/config/_index.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+| [list](/v2.0/reference/cli/influx/config/create)   | List connection configurations.
+| [delete](/v2.0/reference/cli/influx/config/delete  | Delete a connection configuration.
+| [set](/v2.0/reference/cli/influx/config/set)       | Set or update a connection configuration.
+| `-h`, `--help`                                     | Help for the `config` command.
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/_index.md]: Help isn't  a subcommand, it's a flag.

File: content/v2.0/reference/cli/influx/config/create.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+title: influx config create
+description: The 'influx config create' command creates a new InfluxDB connection configuration.
+menu:
+  v2_0_ref:
+    name: influx config create
+    parent: influx config
+weight: 201
+---
+
+The `influx config create` command creates a new InfluxDB connection configuration and stores it in the `config` file (by default, stored at `~/.influxdbv2/config`).
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/create.md]: - ~`config`~ -> `configs`
- ~`~/.influxdbv2/config`~ -> `~/.influxdbv2/configs`

File: content/v2.0/reference/cli/influx/config/create.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+influx config create [flags]
+```
+
+## Flags{
+| Flag                | Description                                                    | Input type  |
+|:----                |:-----------                                                    |:----------: |
+| `-a,`, `--active`   | Set the specified connection to be the active configuration.   |             |
+| `-h`, `--help`      | Help for the `create` command                                  |             |
+| `-n`, `--name`      | (**Required**) Name of the new configuration.                  | string      |
+| `-o`, `--org`       | (Optional) Organization name                                   | string      |
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/create.md]: I don't think you need "(Optional)"

File: content/v2.0/reference/cli/influx/config/delete.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+title: influx config delete
+description: The 'influx config delete' command deletes an InfluxDB connection configuration.
+menu:
+  v2_0_ref:
+    name: influx config delete
+    parent: influx config
+weight: 201
+---
+
+The `influx config delete` command deletes an InfluxDB connection configuration from the `config` file (by default, stored at `~/.influxdbv2/config`).
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/delete.md]: - ~`config`~ -> `configs`
- ~`~/.influxdbv2/config`~ -> `~/.influxdbv2/configs`

File: content/v2.0/reference/cli/influx/config/delete.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+## Usage
+```
+influx config delete [flags]
+```
+
+## Flags
+| Flag                | Description                                                        | Input type  | {{< cli/mapped >}}   |
+|:----                |:-----------                                                        |:----------: |:------------------   |
+| `-h`, `--help`      | Help for the `delete` command                                      |             |                      |
+| `-n`, `--name`      | (**Required**) Name of InfluxDB connection configuration to delete | string      |                      |
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/delete.md]: You can removed the cli/mapped column. None of these map.

File: content/v2.0/reference/cli/influx/config/list.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+title: influx config list
+description: The 'influx config list' command lists all InfluxDB connection configurations.
+menu:
+  v2_0_ref:
+    name: influx config list
+    parent: influx config
+weight: 201
+---
+
+The `influx config list` command lists all InfluxDB connection configurations in the `config` file (by default, stored at `~/.influxdbv2/config`). Each connection configuration includes a url, authentication token, and active setting. An asterisk (`*`) indicates the active configuration.
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/list.md]: - ~`config`~ -> `configs`
- ~`~/.influxdbv2/config`~ -> `~/.influxdbv2/configs`
- ~url~ -> URL

File: content/v2.0/reference/cli/influx/config/set.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+title: influx config set
+description: The 'influx config set' command updates an InfluxDB connection configuration.
+menu:
+  v2_0_ref:
+    name: influx config update
+    parent: influx config
+weight: 201
+---
+
+The `influx config update` command updates information in an InfluxDB connection configuration in the `config` file (by default, stored at `~/.influxdbv2/config`).
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/set.md]: - ~`config`~ -> `configs`
- ~`~/.influxdbv2/config`~ -> `~/.influxdbv2/configs`

File: content/v2.0/reference/cli/influx/config/set.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+
+## Usage
+```
+influx config update [flags]
+```
+
+#### Aliases
+
+`set` , `update` TX
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/set.md]: ~TX~

File: content/v2.0/reference/cli/influx/config/_index.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+influx config [command]
+```
+
+## Subcommands
+| Subcommand                                         | Description   |
+|:----                                               |:-----------   |
+| [create](/v2.0/reference/cli/influx/config/create) | Create a new connection configuration
+| [list](/v2.0/reference/cli/influx/config/create)   | List connection configurations
+| [delete](/v2.0/reference/cli/influx/config/delete  | Delete a connection configuration
+| [set](/v2.0/reference/cli/influx/config/set)       | Set or update a connection configuration
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/_index.md]: These rows need a bars on the end of them to properly close the row.

File: content/v2.0/reference/cli/influx/config/_index.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+
+## Flags
+| Flag               | Description |
+|:----               |:----------- |
+| `-h`, `--help`     | Help for the `find` command
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/_index.md]: Neds a bar on the end.

File: content/v2.0/reference/cli/influx/config/list.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+#### Aliases
+
+`list`, `ls`
+
+## Flags
+| Flag               | Description |
+|:----               |:----------- |
+| `-h`, `--help`     | Help for the `find` command
```
Comment (sanderson) [inline content/v2.0/reference/cli/influx/config/list.md]: Needs a bar on the end.

File: content/v2.0/reference/cli/influx/config/create.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+## Flags{
+| Flag                | Description                                                    | Input type  |
+|:----                |:-----------                                                    |:----------: |
+| `-a,`, `--active`   | Set the specified connection to be the active configuration.   |             |
+| `-h`, `--help`      | Help for the `create` command                                  |             |
+| `-n`, `--name`      | (**Required**) Name of the new configuration.                  | string      |
+| `-o`, `--org`       | Organization name                                              | string      |
+| `u`, `--url`        | (**Required**) Connection URL for the new configuration.       | string      |
```
Comment (kelwang) [inline content/v2.0/reference/cli/influx/config/create.md]: -u here
also, missing -t --token which is also required

File: content/v2.0/reference/cli/influx/config/set.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+`set` , `update`
+
+## Flags
+| Flag                | Description                                                                     | Input type  |
+|:----                |:-----------                                                                     |:----------: |
+| `-a`, `--active`    | Set the specified connection to active                                          |             |
+| `-h`, `--help`      | Help for the `update` command                                                   |             |
+| `-n`, `--name`      | Name for the InfluxDB connection configuration to set or update                 | string      |
+| `-o` , `--org`      | Organization name for the connection configuration                              | string      |
+| `-u`, `--url`       | (**Required**) URL for InfluxDB connection configuration to set or update       | string      |
```
Comment (kelwang) [inline content/v2.0/reference/cli/influx/config/set.md]: missing -t --token here

File: content/v2.0/reference/cli/influx/config/create.md (hunk 18 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/config/create.md]: @kelwang I fixed this on master. Also, `token` is a global flag, so it's listed separately.

**Pull request merged**